ticdc/ticdc-bidirectional-replication.md

```diff
@@ -34,47 +34,117 @@ TiCDC only replicates incremental data changes that occur after a specified time
 
-## Execute DDL
-
-After the bidirectional replication is enabled, TiCDC does not replicate any DDL statements. You need to execute DDL statements in the upstream and downstream clusters respectively.
-
-Note that some DDL statements might cause table structure changes or data change time sequence problems, which might lead to data inconsistency after the replication. Therefore, after enabling bidirectional replication, only the DDL statements in the following table can be executed without stopping the write operations of the application.
-
-| Event | Does it cause changefeed errors | Note |
-|---|---|---|
-| create database | Yes | After you manually execute the DDL statements in the upstream and downstream clusters, the errors can be automatically recovered. |
-| drop database | Yes | You need to manually restart the changefeed and specify `--overwrite-checkpoint-ts` as the `commitTs` of the DDL statement to recover the errors. |
-| create table | Yes | After you manually execute the DDL statements in the upstream and downstream clusters, the errors can be automatically recovered. |
-| drop table | Yes | You need to manually restart the changefeed and specify `--overwrite-checkpoint-ts` as the `commitTs` of the DDL statement to recover the errors. |
-| alter table comment | No |  |
-| rename index | No |  |
-| alter table index visibility | No |  |
-| add partition | Yes | After you manually execute the DDL statements in the upstream and downstream clusters, the errors can be automatically recovered. |
-| drop partition | No |  |
-| create view | No |  |
-| drop view | No |  |
-| alter column default value | No |  |
-| reorganize partition | Yes | After you manually execute the DDL statements in the upstream and downstream clusters, the errors can be automatically recovered. |
-| alter table ttl | No |  |
-| alter table remove ttl | No |  |
-| add **not unique** index | No |  |
-| drop **not unique** index | No |  |
-
-If you need to execute DDL statements that are not in the preceding table, take the following steps:
-
-1. Pause the write operations in the tables that need to execute DDL in all clusters.
-2. After the write operations of the corresponding tables in all clusters have been replicated to other clusters, manually execute all DDL statements in each TiDB cluster.
-3. After the DDL statements are executed, resume the write operations.
+## DDL types
+
+Starting from v7.6.0, to support DDL replication as much as possible in bi-directional replication, TiDB divides the [DDLs that TiCDC originally supports](/ticdc/ticdc-ddl.md) into two types: replicable DDLs and non-replicable DDLs, according to the impact of DDLs on the business.
+
+### Replicable DDLs
+
+Replicable DDLs are the DDLs that can be directly executed and replicated to other TiDB clusters in bi-directional replication.
+
+Replicable DDLs include:
+
+- `CREATE DATABASE`
+- `CREATE TABLE`
+- `ADD COLUMN`: the column must be `not null` or have a `default value`
```

Comment:
```suggestion
- `ADD COLUMN`: the column can be `null`, or has `not null` and `default value` at the same time
```

Reply:
has -> with? I am not sure which is better.

Reply:
![Uploading image.png…]()